packages/p2p/src/components/buy-sell/buy-table-content.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useState, useContext } from 'react';
+import PropTypes from 'prop-types';
+import { Loading } from '@deriv/components';
+import { localize } from 'Components/i18next';
+import Dp2pContext from 'Components/context/dp2p-context';
+import { InfiniteLoaderList } from 'Components/table/infinite-loader-list.jsx';
+import { TableError } from 'Components/table/table-error.jsx';
+import { requestWS } from 'Utils/websocket';
+import { RowComponent, BuySellRowLoader } from './row.jsx';
+
+const BuySellTableContent = ({ is_buy, setSelectedAd }) => {
+    let item_offset = 0;
+    const { list_item_limit } = useContext(Dp2pContext);
+    const [is_mounted, setIsMounted] = useState(false);
+    const [has_more_items_to_load, setHasMoreItemsToLoad] = useState(false);
+    const [api_error_message, setApiErrorMessage] = useState('');
+    const [is_loading, setIsLoading] = useState(true);
+    const [items, setItems] = useState([]);
+
+    useEffect(() => {
+        setIsMounted(true);
```

Comment:
isnt this componentDidMount lifecycle already, why do we need is_mounted here?

Reply:
So we know that we can update this component's state in its `loadMoreItems` function.